spec/aptible/cli/subcommands/db_spec.rb

```diff
@@ -519,6 +519,69 @@ def expect_replicate_database(opts = {})
       expect { subject.send('db:replicate', 'nope', 'replica') }
         .to raise_error(Thor::Error, 'Could not find database nope')
     end
+
+    it 'allows logical replication of a database with --version set' do
+      master = Fabricate(:database, handle: 'master')
+      databases << master
+      replica = Fabricate(:database,
+                          account: master.account,
+                          handle: 'replica')
+
+      dbimg = Fabricate(:database_image,
+                        type: 'postgresql',
+                        version: 10,
+                        docker_ref: 'aptible/postgresql:10')
+
+      expect(subject).to receive(:find_database_image).with('postgresql', 10)
+        .and_return(dbimg)
+
+      op = Fabricate(:operation)
+
+      params = { type: 'replicate_logical', handle: 'replica',
+                 docker_ref: dbimg.docker_ref }
```

Comment:
The field on `DatabaseImage` is `docker_repo`. `docker_ref` is the field on `Operation`. It doesn't appear to be causing any issues since it's only used in the local context but it should probably be changed to avoid issues in the future.

Reply:
Very nice catch, thanks 👍